Makefile

```diff
@@ -129,6 +129,7 @@ dev: globals ## Set Environment to DEV
 	$(eval export SYSTEM_DNS_ZONE_NAME=${DEPLOY_ENV}.dev.cloudpipeline.digital)
 	$(eval export APPS_DNS_ZONE_NAME=${DEPLOY_ENV}.dev.cloudpipelineapps.digital)
 	$(eval export ALERT_EMAIL_ADDRESS?=govpaas-alerting-dev@digital.cabinet-office.gov.uk)
+	$(eval export NEW_ACCOUNT_EMAIL_ADDRESS=${ALERT_EMAIL_ADDRESS})
```

Comment:
Why did we need to set this here? Doesn't look like we've changed the usage elsewhere in the PR.

Reply:
https://github.com/alphagov/paas-cf/blob/3a9e9d0325ec75f5848929243cbd7b90ba44b44b/concourse/scripts/pipelines-cloudfoundry.sh#L140

If this isn't set then we do not do user account creation, which before this PR we did not do in `dev`

Reply:
Is it ever going to not be set now?